processing/data_collection/gazette/spiders/mt_cuiaba.py

```diff
@@ -0,0 +1,54 @@
+import json
+import time
+from dateparser import parse
+from datetime import datetime
+from scrapy import Request
+from scrapy.downloadermiddlewares.retry import RetryMiddleware
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+# based on the website
+FIRST_YEAR = 1967
```

Comment:
Why 1967? Are there gazettes since 1967 available in the site?

Reply:
Yes, it's the first year available:
![gazeta-municial-cuiaba](https://user-images.githubusercontent.com/614500/93263114-adf87200-f77b-11ea-8080-a35db033bb3d.png)

Reply:
Do you think we should go that far or it's better to get only recent gazettes (15 years, for example)?

Reply:
Oh... no no. Let's get all of them. I was asking because I'm quite surprise about that. My initial thought was that was a "magic number". It's quite nice to have gazettes for so past dates